spec/components/provider_interface/user_permission_summary_component_spec.rb

```diff
@@ -46,7 +46,7 @@
                setup_at: nil)
       end
 
-      create(:course, :open_on_apply, provider: allowed_providers.first, accredited_provider: provider)
+      create(:course, :open, provider: allowed_providers.first, accredited_provider: provider)
```

Comment:
We do have the factory open_on_apply and used in many service specs. Is there a plan to remove all of those? Will this be done on "Drop the open_on_apply column when it is not used anymore" card?

Reply:
Yes, I'm in the process of removing `open_on_apply` from all factory call sites at the moment. I've done `SupportInterface`, this is `ProviderInterface` and there is a PR for `CandidateInterface` now too.

When neither the trait nor the column is being used anymore I'll drop the column but not before informing DI

Reply:
Perfect! :)